doc/test-suite-diff.py

```diff
@@ -130,11 +131,11 @@ def end_webpage(html, body, htmlOutFile):
             pngFileOut = os.path.join(path_out, item1, name + '.png')
             pngFile1Out = os.path.join(path_out, item1, name + '.after.png')
             pngFile2Out = os.path.join(path_out, item1, name + '.before.png')
-            print(pngFile1, pngFile2)
+            print(f'Comparing {name}')
```

Comment:
This is actually changing the output of print, so it’s not simply a formatting change

Reply:
Correct, but simply printing the same file name twice with full paths isn't very informative, so it's still a kind of formatting change ;-)